### PR TITLE
Replaced global access of configuration with importing config module.

### PR DESCRIPTION
**Pull request by macrobug**
This helps with testing, decoupling, and makes it easier to enable scenarios such as converting into an installable component.

### CODE REVIEW

File: cfg/dist.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ let config = Object.assign({}, baseConfig, {
       'RETRY_MIN_TIMEOUT': constants.retryMinTimeoutInMiliseconds,
       'RETRY_MAX_TIMEOUT': constants.retryMaxTimeoutInMiliseconds,
       'TICKET_REFRESH_INTERVAL': constants.ticketRefreshIntervalInSeconds,
+      'AAD_INSTANCE': constants.aadInstance,
```
Comment (strand) [inline cfg/dist.js]: Can we extract out the duplicate attributes in this and the other `cfg/*.js` files into a shared object so the structure of this object does not get out of sync?

File: src/config/README.md [hunk 2]
```diff
@@ -1,9 +1,13 @@
 # About this folder
 
 This folder holds configuration files for different environments.
+
 You can use it to provide your app with different settings based on the 
 current environment, e.g. to configure different API base urls depending on 
 whether your setup runs in dev mode or is built for distribution.
+
+***Note that there is a Webpack alias defined in /cfg/base.js which redirects `'config'` to `'${projectRoot}/config/$env'`***
```
Comment (strand) [inline src/config/README.md]: 🤔 Can you specify _why_ the redirection is interesting or surprising? What in the development process made you decide to update the `README.md`?

Comment (macrobug) [reply src/config/README.md]: Good question. The redirection itself is neither interesting or surprising. What was not immediately clear to me is where the redirection alias was defined (note that there are two places for configuration ./cfg/ and ./src/config/).

Also, note that this is not the main README file, but a config specific README file. So, it made sense to me to put the note here to help people understand the config pipeline a bit more.

File: cfg/dist.js (hunk 1 above)
Comment (macrobug) [reply cfg/dist.js]: Looking into it...

File: cfg/test.js [hunk 3]
```diff
@@ -45,17 +45,7 @@ var config = {
...
-      'AUTH_REDIRECT_URI': constants.authRedirectUri,
-      'RETRIES': constants.retries,
-      'RETRY_EXPONENTIAL_BACKOFF_FACTOR': constants.retryExponentialBackoffFactor,
-      'RETRY_MIN_TIMEOUT': constants.retryMinTimeoutInMiliseconds,
-      'RETRY_MAX_TIMEOUT': constants.retryMaxTimeoutInMiliseconds,
-      'TICKET_REFRESH_INTERVAL': constants.ticketRefreshIntervalInSeconds,
-      'AAD_TENANT': constants.aadTenant,
-      'CLIENT_ID': constants.clientId
-    })
+    new webpack.DefinePlugin({constants})
```
Comment (strand) [inline cfg/test.js]: 👏

**Pull request merged**